CHANGELOG.md

```diff
@@ -11,10 +11,11 @@
 # Unreleased
 ## Fixes
 - get_relations_by_pattern for Databricks connection with Unity catalog #768 ([#768](https://github.com/dbt-labs/dbt-utils/issues/768), [#769](https://github.com/dbt-labs/dbt-utils/pull/769))
+- Handle empty strings in slugify macro #773 ([#773](https://github.com/dbt-labs/dbt-utils/issues/773), [#774](https://github.com/dbt-labs/dbt-utils/pull/774))
```

Comment:
We have a merge conflict that we need to resolve in CHANGELOG.md.

I wasn't able to push to the remote for some reason, so I'm going to try to remove these changes in order to resolve the merge conflict.

```suggestion
```